pkg/transform/configmaps/configmaps_test.go

```diff
@@ -1,9 +1,11 @@
-package configmaps
+package configmaps_test
```

Comment:
Why did you change the package? I find it easier to test when the tests are in the same package.

Reply:
If test belongs to `configmaps` I am getting import cycle error in this test.
```
import cycle not allowed in test
package github.com/fusor/cpma/pkg/transform/configmaps (test)
	imports github.com/fusor/cpma/pkg/transform
	imports github.com/fusor/cpma/pkg/transform/configmaps
```

Reply:
I thought about it, but I don't know what would be a good fit for `GenYAML`, maybe `utils/` ?

Reply:
It makes sense to test genYAML in transform since it's part of it!